util/manually-added-ecqm-data.json

```diff
@@ -0,0 +1,329 @@
+[
+  {
+    "eMeasureId": "CMS117v5",
+    "strata": [
+      {
+        "name": "receivedVaccinesOrExposure",
+        "description": "Children who have evidence showing they received recommended vaccines, had documented history of the illness, had a seropositive test result, or had an allergic reaction to the vaccine by their second birthday"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS123v5",
+    "strata": [
+      {
+        "name": "receivedExams",
```

Comment:
Is it worth being more specific here? Maybe `receivedFootExams` or even `receivedVisualPulseAndSensoryFootExams`?